src/services/jsDoc.ts

```diff
@@ -70,7 +71,7 @@ namespace ts.JsDoc {
         const tags: JSDocTagInfo[] = [];
         forEachUnique(declarations, declaration => {
             for (const tag of getJSDocTags(declaration)) {
-                if (tag.kind === SyntaxKind.JSDocTag) {
+                if (tag.kind === SyntaxKind.JSDocTag || tag.kind === SyntaxKind.JSDocInheritDocTag) {

```

Comment:
Don't think you need to modify this function for what you want; use `ts.getJSDocTags` (from `compiler/utilities.ts`) instead, which returns tags of all kinds.